src/App.vue

```diff
@@ -347,6 +347,33 @@ export default Vue.extend({
             if(event.key.toLowerCase() == "contextmenu"){
                 this.appStore.isContextMenuKeyboardShortcutUsed  = true;
             }
+
+            // Handling the notification for doing duplication with drag and drop.
+            // We don't really care if another key is hit along ctrl/meta, we only look that
+            // we are currently in a drag and drop action, and notify the current caret candidate for drop that
+            // the action requires frame duplication.
+            if(this.appStore.isDraggingFrame && (event.ctrlKey || event.metaKey)){
```

Comment:
I always find these key names confusing, but on Mac it is "Option" to drag-copy, and that is altKey in Javascript, not metaKey.

Reply:
Very nice catch ! It's my (very stupid) mistake, even considering the key codes are rightfully used in another part of the code, I should have been more careful! Thanks